manifests/0000_90_kube-controller-manager-operator_05_alert-pdb.yaml

```diff
@@ -11,17 +11,17 @@ spec:
       rules:
         - alert: PodDisruptionBudgetAtLimit
           annotations:
-            message: The pod disruption budget is preventing further disruption to pods because it is at the minimum allowed level.
+            message: The pod disruption budget {{ $labels.namespace }}/{{ $labels.poddisruptionbudget }} is preventing further disruption to pods because it is at the minimum allowed level.
           expr: |
-            kube_poddisruptionbudget_status_expected_pods == on(namespace, poddisruptionbudget, service) kube_poddisruptionbudget_status_desired_healthy
+            max by(namespace, poddisruptionbudget) (kube_poddisruptionbudget_status_expected_pods == kube_poddisruptionbudget_status_desired_healthy)
```

Comment:
Since you're changing how this alert is reporting you'll need to update e2e in https://github.com/openshift/cluster-kube-controller-manager-operator/blob/57aab7b4c200f43847f66195d3777b638ddda65a/test/e2e/operator_test.go#L124 to also drop the `service="kube-state-metrics"` bit too.

Reply:
:+1: